src/content/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy.mdx

```diff
@@ -8,7 +8,7 @@ The following are the specific policies and dates for support of our <InlinePopo
 
 ## iOS [#ios-eol]
 
-The following are the specific policies and dates for support of our iOS agent. Any versions not listed in the following table are no longer supported. Please [update your iOS agent version](/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk) to the [latest release](/docs/release-notes/mobile-release-notes/ios-release-notes).
+The following are the specific policies and dates for support of our iOS agent. Any versions not listed in the following table are no longer supported. Please [update your iOS agent version](/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-ios-agent) to the latest release.
```

Comment:
Second sentence: "We don't support versions not listed in the following table." (passive fix)